pkg/datapath/linux/linux_defaults/mark.go

```diff
@@ -37,6 +37,10 @@ const (
 	// MagicMarkProxyNoIDMask can be used to fetch the proxy-relevant magic
 	// bits from a mark for proxy reply traffic.
 	MagicMarkProxyNoIDMask int = 0xFFFFFEFF
+	// MagicMarkIsProxyEPID can be used in conjunction with MagicMarkProxyMask
+	// to determine whether the mark is indicating that traffic is sourced
+	// from a proxy prior to endpoint policy enforcement.
+	MagicMarkIsProxyEPID int = 0x0800
```

Comment:
The lowest bit is already used with 0x0A00 and 0x0B00 to tell the difference between ingress/egress, where the lowest bit is 0 or 1, respectively.

I carried over the same semantic to "proxy EPID" mark, where ingress uses 0x0800 and egress uses 0x0900. For testing if the mark includes the endpoint ID, masking with 0x0800 is needed, so that the difference between ingress/egress is abstracted away.

Now it turns out that at least for now, we only need the "mark includes EPID" for egress. Use of these patterns is already built in to the Envoy image where the mark is set, but we could re-spin that and make both `MagicMarkProxyEgressEPID` and `MARK_MAGIC_PROXY_EGRESS_EPID` be 0x0800 instead. But then the semantic of the lowest bit in this nibble would be reversed from that of MagicMarkIsProxy...